elba.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "gallais/idris-tparsec"
```

Comment:
Can we call it `tparsec`? This is the name I have used on [the wiki](https://github.com/idris-lang/Idris-dev/wiki/Libraries) and the ipkg file.

Reply:
Apparently not, the [manual says](https://elba.readthedocs.io/en/latest/usage/manifest.html):

> The name must contain a group (i.e. a namespace) and a name, separated by a slash, or else the manifest will fail to parse.

Reply:
We can use a different namespace than your handle, I guess, though

Reply:
Eh. I guess if we have to abide by this format, the least confusing option is to stick to the repo's url.

Reply:
Ah, I can also shorten the name to `gallais/tparsec`, is that fine?

Reply:
Yeah maybe let's leave it same as the url though